File: python/2167.py
k = int(input())
l = input().split()
i = 0
a = 0
v = 0

while i < len(l):
    if a > int(l[i]):
        v = i+1
        break
    a = int(l[i])
    i += 1
     
print(v)

File: dart/2167.dart
import 'dart:io';

void main() {
  int n = int.parse(stdin.readLineSync()!);
  List<String> l = stdin.readLineSync()!.split(" ");
  int r = 0;
  int aux = 0;

  for (int i = 0; i < n; i++) {
    if (aux > int.parse(l[i])) {
      r = i + 1;
      break;
    }

    aux = int.parse(l[i]);
  }

  print(r);
}
